docs/en/logs/using.asciidoc

```diff
@@ -0,0 +1,97 @@
+In live streaming mode, you are not able to choose a different time in the time selector or use the minimap timeline.
+To do either of these things, you need to stop live streaming first.
+// ++ Not sure whether this is correct or not. And what about just scrolling through the display?
+// ++ There may be a bug here, (I managed to get future logs) see https://github.com/elastic/kibana/issues/43361
+
+[float]
+[[logs-highlight]]
+=== Highlight a phrase in the logs stream
+To highlight a word or phrase in the logs stream, click *Highlights* and enter your search phrase.
+// ++ Is search case sensitive?
+// ++ Can you search for multiple phrases together, if so, what's the separator?
+// ++ What about special characters? For example, I notice that when searching for "Mozilla/4.0" which appears as written in my logs, "Mozilla" is highlighted, as is "4.0" but "/" isn't. The string "-" (which appears in the logs as written, quotes and all, isn't found at all. Any significance?
+
+[[logs-event-inspector]]
+=== Inspect log events
+To inspect a log event, hover over it, then click the *View actions for line* icon image:images/logs-action-menu.png[View actions for line icon]. On the menu that opens, select *View details*. This opens the *Log event document details* fly-out that shows the fields associated with the log event.
+
+To filter the logs stream by one of the field values, in the log event details, click the *View event with filter* icon image:images/logs-view-event-with-filter.png[View event icon] beside the field.
+A search filter is automatically added to the logs stream to enable you to filter the entries by this field and value.
+
+[float]
+[[log-view-in-context]]
+=== View log line in context
+To view a certain line in its context (for example, with other log lines from the same file, or the same cloud container), hover over it, then click the *View actions for line* image:images/logs-action-menu.png[View actions for line icon]. On the menu that opens, select *View in context*. This opens the *View log in context* modal, that shows the log line in its context.
+
+[role="screenshot"]
+image::images/logs-view-in-context.png[View a log line in context]
+
+[float]
+[[view-log-anomalies]]
+=== View log anomalies
+
+When the machine learning anomaly detection features are enabled, click *Log rate*, which allows you to
+<<xpack-logs-analysis,use machine learning to detect and inspect anomalies>> in your log data.
+
+[[logs-integrations]]
+=== Integrate with Uptime and APM
+
+To see other actions related to the event, click *Actions* in the log event details.
+Depending on the event and the features you have configured, you may also be able to:
+
+* Select *View status in Uptime* to {uptime-guide}/uptime-app-overview.html[view related uptime information] in the *Uptime* app.
+* Select *View in APM* to {kibana-ref}/traces.html[view related APM traces] in the *APM* app.
```

Comment:
What about adding a screenshot from dev-next here? I think we should be mindful of adding _too_ many screenshots, but this seems like a good spot for one? What do you think?